resource/tasks.json

```
@@ -7831,11 +7831,34 @@
     "BattleQuickFormationSkill3": {
         "algorithm": "JustReturn",
         "action": "ClickRect",
+        "specificRect": [
+            15,
+            650,
+            80,
+            30
```

```suggestion
            15,
            710,
            80,
            10
```
这个要不还是给设到最底下吧，不知道上面的描述会有多长

---

这是因为划太快了短描述点不到：）

---

草

---

> 要不就滑动的postDelay延长，等他回弹之后，再点 不然上面的也不知道多长，就没法点

按照现在的参数，没划到底的时候至少滑动距离是225，点击区域只上移了60，这样之前能点的应该都能点，不会出事（确信

---

合了测测